workflows/ci-simple/versions/0.0.1/docs/ci-simple.md

```diff
@@ -0,0 +1,67 @@
+# ci simple
+
+## Summary
+This CI pipeline builds a docker image using Kaniko, uploads image metadata to CSDP, and tests the image
+
+
+## Inputs/Outputs
+
+#### Parameters
+* REPO (required) - the repository you want to clone (for example: https://github.com/codefresh-io/cli-v2)
```

Comment:
When one is using the New Pipeline wizard, it's a lot easier to guess what is needed for each input if their names start with their context. For example, GIT_REPO_URL instead of REPO, IMAGE_TAG instead of TAG.